easybuild/framework/easyconfig/default.py

```diff
@@ -165,6 +165,7 @@
 
     # OTHER easyconfig parameters
     'buildstats': [None, "A list of dicts with build statistics", OTHER],
+    'options' : [{},"A list of extra options for specific blocks", OTHER],
```

Comment:
@Bart-CER maybe you need https://github.com/hpcugent/easybuild/wiki/Development-guide#adding-specification-options ?
there's an example in the code lower on that same page